### ISSUE TITLE
[Bug]: Key items inventory full

### ISSUE THREAD

**Issue by cristi420** [LukeYui/EldenRingSeamlessCoopRelease#72]
### Discord Username

cristi420#1806

### Describe the issue in detail

I cannot pick up any key items (like coockbooks, great runes, dragonhearts) and if i try to move a key item to chest i will not be able to put it back in the inventory because it is "full". I think everything started when i could not get more Rot (the debuff from death) because i had some on me and a lot got stacked in the chest at the grace. I deleted them one by one and after that it continued to tell me my inventory is full. I can't even get Rot now.

### Steps to recreate the bug

I think everything started when i could not get more Rot (the debuff from death) because i had some on me and a lot got stacked in the chest at the grace. I deleted them one by one and after that it continued to tell me my inventory is full. I can't even get Rot now. We are 3 players, we were in snowy mountains or consecrated snowfields.

### Version

1.2.1 (Latest)

**Comment by Sephralard** [comment]
Me and my friend are having this issue.
For us it started when we picked the "The Flame of Frenzy" enchant, the game gave to my friend like 70 or 80 tiny gray pots (The opening world item), after that the game had the inventory limited. Some key items where directly put in the chest of the grace, and some just dissapear and you don't get the item. We tested a little and it seems to be related with the key item inventory slots, cause we stored the stone keys in order to get an extra slot and we could take an item from the chest, but when you pick an item like a map or a cookbook you are going to waste that slot, my friend lost a wetstone for that bug.
